examples/dgl/ogbn_products_sage_quiver.py

```diff
@@ -233,19 +229,31 @@ def run(args, device, data):
         device = th.device('cpu')
 
     # load ogbn-products data
-    data = DglNodePropPredDataset(name='ogbn-products', root='/data/products')
+    data = DglNodePropPredDataset(name='ogbn-products', root='/workspace/gnn/dataset/')
```

Comment:
You could keep the original root or move it to args

Reply:
Fixed